src/main/java/org/apache/datasketches/theta/SetOperationBuilder.java

```diff
@@ -207,11 +207,11 @@ public SetOperation build(final Family family, final WritableMemory dstMem) {
       }
       case A_NOT_B: {
         if (dstMem == null) {
-          setOp = new HeapAnotB(bSeed);
+          setOp = new AnotBimpl(bSeed);
         }
         else {
-          throw new SketchesArgumentException(
-            "AnotB is a stateless operation and cannot be persisted.");
+          throw new SketchesArgumentException( //TODO we should be able to do this now.
```

Comment:
is this TODO comment still relevant?

Reply:
Removed the TODO, but still not allowing AnotB to be persisted.